rct229/rulesets/ashrae9012019/section19/section19rule21.py

```diff
@@ -0,0 +1,498 @@
+
+                return UNDETERMINED_MSG
+
+            def rule_check(self, context, calc_vals=None, data=None):
+                climate_zone = calc_vals["climate_zone"]
+                OA_fraction_b = calc_vals["OA_fraction_b"]
+                supply_airflow_b = calc_vals["supply_airflow_b"]
+                ER_modeled_b = calc_vals["ER_modeled_b"]
+                exception_1_applies = calc_vals["exception_1_applies"]
+                exception_2_applies = calc_vals["exception_2_applies"]
+                exception_6_applies = calc_vals["exception_6_applies"]
+                exception_7_applies = calc_vals["exception_7_applies"]
+                ER_not_req_for_heating_sys = calc_vals["ER_not_req_for_heating_sys"]
+                sys_type_heating_only = calc_vals["sys_type_heating_only"]
+                all_lighting_space_types_defined = calc_vals[
+                    "all_lighting_space_types_defined"
+                ]
+                all_ventilation_space_types_defined = calc_vals[
+                    "all_ventilation_space_types_defined"
+                ]
+                ER_modeled_p = calc_vals["ER_modeled_p"]
+
+                return (
+                    (
+                        OA_fraction_b >= OA_fraction_b_70
+                        and supply_airflow_b >= SUPPLY_AIRFLOW_5000CFM
+                        and (
+                            # CASE 1
+                            (ER_modeled_b)
+                            or
+                            # CASE 2
+                            (not ER_modeled_b and exception_1_applies)
+                            or
+                            # CASE 3
+                            (not ER_modeled_b and exception_2_applies)
+                            or
+                            # CASE 4
+                            (
+                                not ER_modeled_b
+                                and ER_not_req_for_heating_sys
+                                and sys_type_heating_only
+                            )
+                            or
+                            # CASE 5
+                            (not ER_modeled_b and exception_6_applies)
+                            or
+                            # CASE 6
+                            (not ER_modeled_b and exception_7_applies)
+                        )
+                    )
+                    or (
+                        (
+                            OA_fraction_b < OA_fraction_b_70
+                            or supply_airflow_b < SUPPLY_AIRFLOW_5000CFM
+                        )
+                        and
+                        # CASE 9
+                        (not ER_modeled_b)
+                        or
+                        # CASE 10
+                        (ER_modeled_b)
```

Comment:
I think it is not necessary to add these two cases - essentially , it means pass regardless of ER_modeled or not.

Reply:
Addressed.